### ISSUE TITLE
IA: Create a short intro to IA work

### ISSUE THREAD

**Issue by kristinashu** [mozilla/foundation.mozilla.org#4417]
We need a quick intro to all the IA work in order to share our progress with all staff.

This ticket is blocking - IA: shareout of the report to all staff #4302

### Todo
- [ ] Update [this deck](https://docs.google.com/presentation/d/11CK6yYXx8iCSsTG3Lc89-2k4Eiuy21zMoJIrmG7vjfY/edit#slide=id.g25275a8168_0_260) so it is up-to-date and relevant for all staff

cc @taisdesouzalessa

**Comment by beccaklam** [comment]
@kristinashu @taisdesouzalessa I revamped the[ first bit of our original deck](https://docs.google.com/presentation/d/19etCN1uGLcLmB50-uV-srsWKEAhlwlR_bc0dLE1uW1g/edit?usp=sharing) (sharing a copy before I edit the original). Let me know what you think? I thought most of the context setting content at the beginning was still good, just took out some old dates. Then I condensed the analytic recommendations to one page.
